lib/tools/test/emem_SUITE.erl

```diff
@@ -138,8 +138,7 @@ end_per_suite(Config) when is_list(Config) ->
 
 live_node(Config) when is_list(Config) ->
     {ok, EmuFlag, Port} = start_emem(Config),
-    Nodename = mk_nodename(Config),
```

Comment:
This SUITE does not compile as the `Nodename` variable is used further down in this function.

Reply:
My bad, I did not intend to change this file at all, but asked @bjorng whether it can be removed altogether. I was not able to find any way to actually run these tests. They are always skipped, because `proplists:get_value(data_dir, Config)` (line 100) returns the name of the directory that does not exist.
I was also not able to find any mentions of "emem" utility except for `c_src` folder that builds it. I could not find any documentation what the tool does either.

How can I run these tests, and how do I run the tool? It felt abandoned, so I wonder whether I can leave test case as-is, and then make another PR with the tool removed completely, or documented/explained.

Reply:
The emem tool is a "memory tracer" that was used a long time ago, but the code in the emulator is still there and it can be enabled using `+Mit IP:port`. Though since it seems like it has not been tested in a long time I would be very surprised if it will worked.

@rickard-green do you have any idea where the emem_SUITE_data has gone? It seems like it was not included in the initial import from clearcase to git? I suppose that we can remove it all without anyone missing it?

Reply:
@rickard-green says that we can remove it, so a PR would be welcome.

Reply:
My pleasure - https://github.com/erlang/otp/pull/5591